Makefile

```diff
@@ -70,10 +65,6 @@ install-tools:
 	  github.com/pavius/impi/cmd/impi \
 	  github.com/tcnksm/ghr
 
-.PHONY: otelcontribcol
-otelcontribcol:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_$(GOOS)_$(GOARCH) $(BUILD_INFO) ./cmd/otelcontribcol
```

Comment:
why no `otelcontribcol` anymore? we have it in core as `otelcol`, looks like a reasonable target

Reply:
It is the same as binaries. Exactly same mistake in core. Why do we need 2 targets that do the same?

Reply:
binaries is nicer because it is the same between the repos ;)

Reply:
I usually run `make otelcol` to get `otelcol` binary :) 
`make build` would be a good option as well, what do you think?
And we still have `make docker-otelcol` :)

Reply:
Happy to review a PR to use otelcolcontrib and remove binaries, probably same in code

Reply:
Make build is better I think